### DISCUSSION TITLE
Economy Balance Log

### DISCUSSION THREAD

**Discussion by Ash10744** [Zrips/CMI#2049] [Feature Requests]
**Description of feature/improvement request:**

A command that's adds onto the economy commands you have that basically displays all the transactions the player has done.
(/cmi money log (playername/self)

---
### Detailed explanation of what it should do, when and in what situation: ###

Basically it will allow people to track scamming if it ever occurred, and also allow players to provide follow up proof of a transaction they have done.

### Basic explanation as to why this feature is needed and/or useful to have: ###

More moderation tools within the server, and allows for players to see their previous transactions.

Thank you for reading <3

**Comment by WolverStones** [comment]
Basically, it already exists, it's the moneyLog.yml file, and you need to enable   # Set to true if you want to log money transfers between players
  LogEnabled: true in the configuration file

**Comment by Ash10744** [comment]
Is there an in game command to display it in game? That's what I wanted to request if not

**Comment by mrfloris** [comment]
The amount of data to display in the chat is ridiculous for even small servers - I am not saying it's impossible, but it gives a skewed overview versus what a stand alone .log file can offer. It would require extensive search/query tools with additional data just to properly display something in a human readable format ingame with pagination that isn't unreasonable. 

today's .log file for a server with 10 players is already 50,000 lines long. It's much more efficient to use grep and regex from a command line interface than something in-game. Especially because when you generally need this it's to figure out if there's abuse by a player, or to fix malicious things like /sellall from dupes, etc. 

While it's nice to see a simple history overview, it's not simple; there's loads of data. Just my opinion.
